### PR TITLE
Increases likeliness of extended on MRP

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
makes extended more likely on mrp

## Why It's Good For The Game

after playing mrp for a bit, i realize it's the same as LRP was now... about rolling muh antag and getting muh valids, with most rounds lasting the LRP average of 40-50 minutes.  Making extended this much more likely will make the antag-hunters gravitate towards Golden more, and allow for long, calm rounds to occur. Extended rounds are also an excellent way to weed out the shitters. Let's get those nice long rounds again

## Changelog
:cl:
config: extended is a lot more common on sage. if you really want to roll that antag, golden is your best bet
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Can you maybe increase secret extended as well and have them both at 5 or 6? (Also there is no way this will pass the vote lmao)

Also 9 is wayyyy to high

**Comment by Flakkyboo** [comment]
> 
> 
> Can you maybe increase secret extended as well and have them both at 5 or 6? (Also there is no way this will pass the vote lmao)
> 
> Also 9 is wayyyy to high

exactly what i said 6 is a good number 9 is just salt.

**Comment by St0rmC4st3r** [comment]
> Can you maybe increase secret extended as well and have them both at 5 or 6?

I second that. Secret extended should be as common as regular extended. This will ultimately be better than just explicit greenshifts happening very often.

I understand that the goal of the PR is to make greenshifts as common as traitors, but it will be better having secret extended more often than twice as aften as Wizard.

**Comment by zeskorion** [comment]
> Can you maybe increase secret extended as well and have them both at 5 or 6? (Also there is no way this will pass the vote lmao)
> 
> Also 9 is wayyyy to high

Fuck the vote.

**Comment by Isy232** [comment]
I honestly agree with this, its nice to be guaranteed a long round so you know you wont have to leave part way through it. Its annoying to try to build a shuttle or some other cool stuff and then just have medbay go boom and red alert shuttle get called.

Extended rounds can also be extremely chaotic, they just do not have any roundstart antags.

**Comment by yorii** [comment]
I think it's a nice change but 9 is a bit too high isn't it?

**Comment by TeomanTheGreat** [comment]
Yeah 9 is waaaaay too high make it like 6 or 7

File: config/Sage/game_options.txt [hunk 1]
```diff
@@ -99,7 +99,7 @@ PROBABILITY TRAITORCHAN 5
 PROBABILITY CHANGELING 5
 PROBABILITY REVOLUTION 4
 PROBABILITY NUCLEAR 4
-PROBABILITY EXTENDED 4
+PROBABILITY EXTENDED 9
```
Comment (Crossedfall) [inline config/Sage/game_options.txt]: Two things. One, keep the list ordered. Two, I agree with the others, 9 is way too high. I think 7 would likely be a bit better.

**Review by Crossedfall — CHANGES_REQUESTED** [review]

**Comment by ike709** [comment]
@Crossedfall Config.

**Review by Crossedfall — APPROVED** [review]
Config changes applied

**Pull request merged**